lib/loader.js

```diff
@@ -26,86 +23,86 @@ function dispatch(handlers, req, res, done) {
   next();
 }
 
-function loadDirectory(dirPath, prefix, app, interceptorsMap, options) {
-  var currentFolder = dirPath + (prefix || '');
-  var files = fs.readdirSync(currentFolder);
-
-  files.forEach(function (filepath) {
-    var stats = fs.statSync(currentFolder + '/' + filepath);
-
-    if (stats.isDirectory()) {
-      loadDirectory(dirPath, prefix + '/' + filepath, app, interceptorsMap, options);
-    }
-    else {
-      loadFile(currentFolder + '/' + filepath, prefix, app, interceptorsMap, options);
-    }
-  });
-}
-
-function loadFile(file, prefix, app, interceptorsMap, options) {
-  if (dotFileMatch.test(file)) {
-    debug('Ignoring this file', file);
-    return;
-  }
-
-  try {
-    loadRouteObj(app, prefix, file, interceptorsMap);
-  } catch (e) {
-    debug('Error autoloading routes', e, file);
-    if (options.throwError) {
-      throw e;
-    }
-  }
-}
-
 function loadRouteObj(app, prefix, file, interceptorsMap) {
   if (!file.endsWith('.js')) {
-    debug('Ignoring file because it doesn\'t end with .js', prefix, file);
+    debug("Ignoring file because it doesn't end with .js", prefix, file);
     return;
   }
 
-  var mod = require(file);
-  var routeList = [].concat(mod.autoroute || mod);
+  // eslint-disable-next-line global-require, import/no-dynamic-require
+  const mod = require(file);
+  const routeList = [].concat(mod.autoroute || mod);
 
   if (routeList.length === 0) {
-    debug('Couldn\'t find route object for file. Does not expose route api.', file);
+    debug("Couldn't find route object for file. Does not expose route api.", file);
     return;
   }
 
-  routeList.forEach(function (route) {
-    var args = [prefix + (route.path.charAt(0) === '/' ? '' : '/') + route.path];
-    args.push(function (req, res, next) {
-      var interceptorFlags = req.interceptorFlags || [];
+  routeList.forEach((route) => {
+    let args = [prefix + (route.path.charAt(0) === '/' ? '' : '/') + route.path];
```

Comment:
change `'` to `"`?

Reply:
I think I was using `single quote` as the default quote mark, but `eslint` and `prettier` formats it automatically to `double quotes` because there is a `single quote` in the string. e.g.

```
'there\'s a single quote inside.` -> "there's a single quote inside."
```